lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1420,7 +1420,14 @@ static bool ParseSILArgs(SILOptions &Opts, ArgList &Args,
   }
 
   if (const Arg *A = Args.getLastArg(options::OPT_sanitize_EQ)) {
-    Opts.Sanitize = parseSanitizerArgValues(A, Triple, Diags);
+    Opts.Sanitize = parseSanitizerArgValues(
+        A, Triple, Diags,
+        /* sanitizerRuntimeLibExists= */[&](const StringRef libName) {
+
+          // The driver has checked the existance of the library
```

Comment:
Typo: "existence". Also, should this be a default argument to `parseSanitizerArgValues` then?